test/Microsoft.Azure.WebJobs.Host.UnitTests/AttributeClonerTests.cs

```diff
@@ -64,6 +65,13 @@ public class Attr4: Attribute
             [AutoResolve]
             public string AutoResolve { get; set; }
         }
+        
+        // Test with DefaultValue.MemberName
+        public class Attr5 : Attribute
+        {
+            [AutoResolve(Default = DefaultResolveValue.MemberName)]
```

Comment:
Yes, it's method. I'll rename.  I'm just taking what the exsiting usage is. 

---
In reply to: [113582170](https://github.com/Azure/azure-webjobs-sdk/pull/1124#discussion_r113582170) [](ancestors = 113582170)